contrib/builder/rpm/ppc64le/README.md

```diff
@@ -0,0 +1,5 @@
+# `dockercore/builder-rpm`
+
+This image's tags contain the dependencies for building Docker `.rpm`s for each of the RPM-based platforms Docker targets.
+
+To add new tags, see [`contrib/builder/rpm/ppc64le` in https://github.com/docker/docker](https://github.com/docker/docker/tree/master/contrib/builder/rpm/ppc64le), specifically the `generate.sh` script, whose usage is described in a comment at the top of the file.
```

Comment:
Maybe we could move the README files for all the builders up a level and consolidate?